Missed fixes

KillerOfPie opened the pull request.

KillerOfPie committed:
Fix for missing updates

KillerOfPie committed:
Fix for missing updates

KillerOfPie committed:
Added directions setting and needed methods

*unused atm will be implemented later

SparklingComet commented on src/main/java/org/shanerx/tradeshop/TradeShop.java:
```diff
@@ -94,47 +103,58 @@ public void onEnable() {
+	
+	private void addDirections()
+    	{
+        ArrayList<Material> Allowed = new ArrayList<>();
```
Please use lowerCamelCase for non constant variables.
(I'll change it this time.)

SparklingComet reviewed (changes requested):
Looks good, but:

// EDIT: Fixed in commit below VVV

SparklingComet committed:
Fix variable names

SparklingComet commented on src/main/java/org/shanerx/tradeshop/TradeShop.java:
```diff
@@ -94,47 +103,58 @@ public void onEnable() {
+	private void addDirections()
+    	{
+        ArrayList<Material> allowed = new ArrayList<>();
+        AllowedOld.addAll(Arrays.asList(new Material[] {BlockFace.DOWN, BlockFace.WEST, BlockFace.SOUTH, BlockFace.EAST, BlockFace.NORTH, BlockFace.UP}));
```
This variable isn't defined anywhere, at least from what I can see. a variable of type List<Material> was defined as a local variable inside #addMaterials(). Perhaps you want to make it a global constant field?

KillerOfPie committed:
Fixed the non declared variable

I wrote that in gitHub and forgot to change what i copied to the proper variable

SparklingComet commented on src/main/java/org/shanerx/tradeshop/TradeShop.java:
```diff
@@ -147,7 +147,7 @@ private void addMaterials()
     	{
         ArrayList<Material> allowed = new ArrayList<>();
-        AllowedOld.addAll(Arrays.asList(new Material[] {BlockFace.DOWN, BlockFace.WEST, BlockFace.SOUTH, BlockFace.EAST, BlockFace.NORTH, BlockFace.UP}));
+        allowed.addAll(Arrays.asList(new Material[] {BlockFace.DOWN, BlockFace.WEST, BlockFace.SOUTH, BlockFace.EAST, BlockFace.NORTH, BlockFace.UP}));
```
Any chance this can be made a (public) final global field?

SparklingComet replied:
No point in declaring a constant inside a function...

KillerOfPie committed:
Added missing/incorrect things

- Final

SparklingComet commented:
Merged and closed.
